### ISSUE TITLE
SMP Server missing temple/ structure at spawn

### ISSUE THREAD

**Issue by N757QQ** [Beyond-Reality/Beyond-Reality-Divergence#149]
As the title says.  I have searched through, and found that there is a command to generate the structure as admin, but cannot actually find the command to do so.   I assume this is a config issue, but have been unable to locate the config files for the structure mod, and am not familiar with the rcst files enough to attempt looking through.

Sorry for a most likely simple issue, but thanks for any help.

**Comment by Dominance** [comment]
Um so the temple is not spawning at all? First I have ever noticed that. Are you moving the world spawn with plugins by any chance? The command is /#gen Temple4.8 once done you can view where it will be placed if you like do /#confirm. Please Follow up with this as the temple not spawning is a serious issue that I have not came across before and cannot replicate.

**Comment by N757QQ** [comment]
Correct, it is not spawning at all.  I initially started the server on a second computer, and it had the temple in that iteration, but due to it having windows as the OS, the server was having issues with memory, CPU usage, and a few other things,

So, I reformatted to MineOS, and upon importing the server and going through the Launchserver, and then eula confirm, and  actually launching, I created a new world, as the old was brand new as well.

It did not have a temple at all.  I /op'd myself and the other player as well to look around for it, and no luck.  I have deleted and reinitiated the server several times without success.  This started in 1.2.4 and went on to 1.2.5

I have not touched any configs, so that may be the issue, I am very new to server administration, and may be making a mistake in setup.   I am reinitializing the server now, and will let you know if the /#gen command works.

When I initiate a server on my 'gaming' pc that I normally run the client side on, I get a temple.  I am going to try to import that server setup as well, if all else fails.   I will report back asap, I am working on this now.

Thanks,

**Comment by N757QQ** [comment]
Okay, the latest version I still had installed server side was without a temple.  The command did work, and I now have the temple, so all is well.

I do not know why it did not gen, but again, I am new, and it certainly may be an issue I caused,

Thanks

**Comment by Dominance** [comment]
Very strange well I have tried to virt box Cent0s and made a new world with the temple generating. I also did it directly on windows 10 with no issues. I cannot replicate this issue but at least the generation command worked for you :)

**Comment by N757QQ** [comment]
I have sorted this issue.  I have now produced a server with the temple.  It was an issue with MineOS having a properties file config section.  When the server generated its own .properties file, the MineOS settings override it, and theirs has enable structures set to false.  Fixing that upon install, then initiating the server fixed this issue.

Had a feeling it was on my end, but in case anyone else has this issue.
